onnxruntime/test/providers/cpu/model_tests.cc

```diff
@@ -25,6 +52,12 @@
 
 extern std::unique_ptr<Ort::Env> ort_env;
 
+#define ASSERT_ORT_STATUS_OK(function)   \
+  do {                                   \
+    OrtStatus* _tmp_status = (function); \
+    ASSERT_EQ(_tmp_status, nullptr);     \
```

Comment:
is it possible to include the status code and message in the assertion failure output?